core/ts/Poll.ts

```diff
@@ -28,8 +27,13 @@ import {
   IJsonTCommand,
 } from "maci-domainobjs";
 
-import { MaciState } from "./MaciState";
-import { packTallyVotesSmallVals, packSubsidySmallVals } from "./utils/utils";
+import assert from "assert";
+
+import type { MaciState } from "./MaciState";
+import type { PathElements } from "optimisedmt";
```

Comment:
I can see this is the only import from `optimisedmt`, given we already import `IncrementalQuinTree` from `crypto` (which is exported from `optimisedmt`) should we maybe add one extra export in `crypto` so we don't have to add `optimisedmt` as dependency here in `core`? if you notice all over the codebase we always import `IncrementalQuinTree` directly from `crypto` and not `optimisedmt`